src/pagination/styled-components.js

```diff
@@ -38,5 +42,17 @@ export const DropdownMenu = styled(StyledList, ({$theme}) => ({
 
 export const DropdownButton = styled(StyledBaseButton, ({$theme}) => ({
   color: $theme.colors.black,
-  minWidth: `calc(${$theme.sizing.scale1600} + ${$theme.sizing.scale400})`,
+}));
+
+// The following are meant to be used internallt and not exported to external customers
+
+export const SelectInput = styled(StyledInput, {display: 'none'});
+export const SelectTag = styled(StyledTag, ({$theme}) => ({
+  ...$theme.typography.font300,
+  color: $theme.colors.black,
+  // Hacking to get around not being able to override borders
```

Comment:
This is a side effect of not being able to override `InputContainer` in Select mode.